libnavigation-core/src/main/java/com/mapbox/navigation/core/MapboxNavigation.kt

```diff
@@ -511,18 +529,44 @@ class MapboxNavigation(
          * Use [NavigationOptions.toBuilder] to easily customize selected options.
          */
         @JvmStatic
-        fun defaultNavigationOptions(context: Context): NavigationOptions {
-            return NavigationOptions.Builder(
-                NONE_SPECIFIED,
-                ROUNDING_INCREMENT_FIFTY,
-                DEFAULT_NAVIGATOR_POLLING_DELAY,
-                MapboxDistanceFormatter(
-                    context.applicationContext,
-                    null,
-                    UNDEFINED,
-                    ROUNDING_INCREMENT_FIFTY
+        fun defaultNavigationOptions(context: Context, accessToken: String?): NavigationOptions {
+            val tilesUri = URI("https://api-routing-tiles-staging.tilestream.net")
+            val tilesVersion = "2019_04_13-00_00_11"
+            return NavigationOptions.Builder()
+                .timeFormatType(NONE_SPECIFIED)
+                .roundingIncrement(ROUNDING_INCREMENT_FIFTY)
+                .navigatorPollingDelay(DEFAULT_NAVIGATOR_POLLING_DELAY)
+                .distanceFormatter(
+                    MapboxDistanceFormatter(
+                        context.applicationContext,
+                        null,
+                        UNDEFINED,
+                        ROUNDING_INCREMENT_FIFTY
+                    )
                 )
-            ).build()
+                .onboardRouterConfig(
+                    MapboxOnboardRouterConfig(
+                        obtainOfflineTilesDir(tilesUri.host, tilesVersion),
+                        null,
+                        null,
+                        2,
+                        Endpoint(
+                            tilesUri.toString(),
+                            tilesVersion,
+                            accessToken ?: "",
+                            "MapboxNavigationNative"
+                        )
+                    )
+                ).build()
+        }
+
+        private fun obtainOfflineTilesDir(host: String, version: String): String {
+            val offlineDir = Environment.getExternalStoragePublicDirectory("Offline")
```

Comment:
Do we want to default to external storage? I'd rather not, and default to internal storage (if at all 👀 comment above) especially to avoid permissions management. What about 👇?

```kotlin
private fun obtainOfflineTilesDir(context: Context, host: String, version: String): String {
    return File(context.filesDir, "Offline/$host/$version").absolutePath
}

Reply:
Sounds good to me, I just looked at the old impl and that was the first reference I found.